config/javadoc.yaml

```
@@ -29,3 +29,8 @@ htmlVolume:
     secretName: javadoc
     shareName: javadoc
     readOnly: true
+
+replicaCount: 2
+
+nodeSelector:
+  agentpool: arm64small
```

@dduportal @smerle33 should I keep this node pool selection for `javadoc`? (cf https://github.com/jenkins-infra/helpdesk/issues/3584)

Or should I switch it to `x86medium`?
```suggestion
  agentpool: x86medium
```